ExilenceNextApp/tsconfig.json

```diff
@@ -18,7 +18,10 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react"
+    "jsx": "react",
+    "noFallthroughCasesInSwitch": true,
+    "noImplicitAny": false,
```

Comment:
I dislike the noImplicitAny, but I guess we need it atm :D

Reply:
Yeah for the time being I went with `noImplicitAny` due to `react-table` as it seemed to keep biting itself with DOM attributes type-wise 🤔

Reply:
Thats fine, I didnt bother too much with the types in `TableWrapper` as I was pretty eager to get it finished